src/mpg123.gyp

```diff
@@ -70,49 +81,72 @@
         {
             'target_name': 'compat',
             'type': 'static_library',
-            'include_dirs': [
-                'util',
-                'mpg123/src',
-                'mpg123/src/compat',
-                'mpg123/src/libmpg123',
-                'config/<(OS)/<(target_arch)'
-            ],
             'defines': [
                 'PIC',
                 'NOXFERMEM',
                 'HAVE_CONFIG_H'
             ],
-            'direct_dependent_settings': {
-                'include_dirs': [
-                    'util',
-                    'mpg123/src',
-                    'mpg123/src/compat',
-                    'mpg123/src/libmpg123',
-                    'config/<(OS)/<(target_arch)'
-                ]
-            },
             'sources': [
                 'mpg123/src/compat/compat.c',
                 'mpg123/src/compat/compat_str.c'
-            ]
+            ],
+            'conditions': [
```

Comment:
For the record, these conditions probably could be done better to be less repetitive, but I can't test cross-platform.